django_app/redbox_app/redbox_core/views/misc_views.py

```diff
@@ -1,14 +1,18 @@
 import logging
+import os
 from http import HTTPStatus
 
 from django.conf import settings
 from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
 from django.views.decorators.http import require_http_methods
 from django.views.generic.base import RedirectView
+from dotenv import load_dotenv
 
 from redbox_app.redbox_core.models import Chat
 
+load_dotenv()
```

Comment:
put 

```python
ALLOW_SIGNUPS = env.bool("ALLOW_SIGNUPS")
```

in the settings.py and reference it like

```python
from django.conf import settings

if settings.ALLOW_SIGNUPS:
    whatever

```

Reply:
Much cleaner - thank you